src/lib.rs

```diff
@@ -110,6 +77,57 @@ extern crate serde_derive;
 #[macro_use]
 mod macros;
 
+// ############################################################################
+// Public API
+// ############################################################################
+pub use self::{
+    chain::quorum_count,
+    error::{InterfaceError, RoutingError},
```

Comment:
InterFaceError is also unused I think, maybe good to look (next PR) at refactoring Error as well.

Reply:
`InterfaceError` is used in some of the public methods of `Node` ([e.g.](https://github.com/maidsafe/routing/blob/42760af38d173a623f86c37b1bdf5b6937618e06/src/node.rs#L245)), so I would keep it exposed for now.